diff -r d687864489da CMakeLists.txt
--- a/CMakeLists.txt	Sat Jan 18 15:17:05 2014 +0700
+++ b/CMakeLists.txt	Sun Apr 06 18:49:36 2014 +0200
@@ -140,9 +140,9 @@
 
-set( CMAKE_CXX_FLAGS "" )
+set( CMAKE_CXX_FLAGS "-std=gnu++11" )
 set( CMAKE_C_FLAGS "" )
-set( CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -funroll-loops -fopenmp -std=gnu++0x" )
-set( CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -std=gnu++0x" )
-set( CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -std=gnu++0x" )
+set( CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -funroll-loops -fopenmp -std=gnu++11" )
+set( CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -std=gnu++11" )
+set( CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -std=gnu++11" )
 
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/Sources
                      ${CMAKE_CURRENT_BINARY_DIR}
