diff -r d687864489da CMakeLists.txt
--- a/CMakeLists.txt	Sat Jan 18 15:17:05 2014 +0700
+++ b/CMakeLists.txt	Sun Apr 06 18:49:36 2014 +0200
@@ -105,11 +105,11 @@
 # Determine Photivo version with mercurial. Use current date if mercurial is unavailable.
   find_program( MERCURIAL hg )
   if( MERCURIAL )
-    execute_process( COMMAND hg branch OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE BRANCH )
-    execute_process( COMMAND hg log --limit 1 --branch ${BRANCH} --template {date|shortdate}
+    execute_process( COMMAND hg -R ${PROJECT_SOURCE_DIR} branch OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE BRANCH )
+    execute_process( COMMAND hg -R ${PROJECT_SOURCE_DIR} log --limit 1 --branch ${BRANCH} --template {date|shortdate}
                      OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE REVDATE )
-    execute_process( COMMAND hg identify --id OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE REVISION )
-    execute_process( COMMAND hg identify -n OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE PACKREVISION )
+    execute_process( COMMAND hg -R ${PROJECT_SOURCE_DIR} identify --id OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE REVISION )
+    execute_process( COMMAND hg -R ${PROJECT_SOURCE_DIR} identify -n OUTPUT_STRIP_TRAILING_WHITESPACE OUTPUT_VARIABLE PACKREVISION )
     set( APPVERSION "${REVDATE} (rev ${REVISION})" )
     string( REGEX REPLACE [+] "" REVISION ${REVISION} )
     string( REGEX REPLACE [+] "" PACKREVISION ${PACKREVISION} )
@@ -140,9 +140,9 @@
 
 set( CMAKE_CXX_FLAGS "" )
 set( CMAKE_C_FLAGS "" )
-set( CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -funroll-loops -fopenmp -std=gnu++0x" )
-set( CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -std=gnu++0x" )
-set( CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -std=gnu++0x" )
+set( CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -funroll-loops -fopenmp -std=gnu++11" )
+set( CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -std=gnu++11" )
+set( CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -std=gnu++11" )
 
 include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/Sources
                      ${CMAKE_CURRENT_BINARY_DIR}
